### PR TITLE
Lang basics tweaks

### PR DESCRIPTION
**Pull request by AssemblyZig**
Providing more direction for the first exercise, and poking at "wibbles" along the way.

### CODE REVIEW

File: labs/Language/Basics.md [hunk 1]
```diff
@@ -216,6 +216,11 @@ import labs::ProjectEuler::ProjectEuler as PE hiding (inc, inc1001)
 Cryptol's module system also supports parameters, but that is covered
 in a later lab.
 
+### File-only commands
+
+It's worth noting that there are a very few Cryptol commands
+that can only be used in a file, *not* interactively in the interpreter.  The most common of these are `module` and `import`.  
+
```
Comment (weaversa) [inline labs/Language/Basics.md:223]: Well, why not just add `private` and `property` to the list?

Comment (AssemblyZig) [reply labs/Language/Basics.md:223]: Sure.  Are they that common?

Comment (weaversa) [reply labs/Language/Basics.md:223]: Yes. Especially `property`. It shows up in _almost_ every lab.

File: labs/Language/Basics.md [hunk 2]
```diff
@@ -359,29 +364,20 @@ makes heavy use sequences, with the occasional tuple and `Integer`
 thrown in.
 
 **EXERCISE**: The Cryptol interpreter command `:type` (or `:t` for
-short) is very useful for helping understand types. Use this command
-in the interpreter to discover the types of the following
-variables.
+short) is very useful for helping understand types. Use the `:type` command
+in the interpreter to determine the types of the variables defined below.
+(*Recall that these variables have been instantiated as part of `:load`ing this module.*)
```
Comment (weaversa) [inline labs/Language/Basics.md:369]: `:module`ing this module?

as part of loading this module with `:module`?

File: labs/Language/Basics.md [hunk 3]
```diff
@@ -572,7 +570,7 @@ labs::Language::Basics> increment 10
    `add 1` takes a 32-bit bitvector and returns a 32-bit
     bitvector. When it is applied to a 32-bit bitvector it adds one to
     that bitvector. Other examples to illustrate partial application:
-  * Really `addUncurried` is a function of one argument. That argument
+  * `addUncurried` is really a function of one argument. It happens that that argument
```
Comment (weaversa) [inline labs/Language/Basics.md:573]: that that -> that

File: labs/Language/Basics.md [hunk 4]
```diff
@@ -648,7 +646,7 @@ Upon reloading this file, we would see:
 ```
 
 Now we get two error messages. One is complaining about the input
-type, and one about the output type.
+type,the other about the output type.
```
Comment (weaversa) [inline labs/Language/Basics.md:649]: ,the
->
, the

**Pull request merged**